storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/query/Query.java

```diff
@@ -103,9 +102,7 @@ public String toString() {
     /**
      * Builder for {@link Query}
      */
-    public static class Builder {
-
-        Uri uri;
+    public static final class Builder {
```

Comment:
`class CustomBuilder exteds Query.Builder {
   CompleteBuilder uri(java.net.URI javaUri) {
      return(javaUri.toString());
   }
}`
and then
storIOContentResolver()
                .get()
                .cursor()
                .withQuery(new Query.CustomBuilder()
                        .uri(existJavaUri)
                        .build())
`
Of cause, it is far-fetched example, but are you sure that it will be not necessary or just want to hide all except really important?